### DISCUSSION TITLE
Play sound when generation completes

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#588] [Ideas]
lshqqytiger's fork in Windows played a sound if you had "notification.mp3" in the same folder as "webui.sh".

This doesn't seem to work for vladmandic's fork, at least in Ubuntu 22.

Also, it'd be nice if a small beep from the console (perhaps using the same file as above, since PC speaker is default blacklisted in ubuntu) could happen after the "Model loaded in ..." message to indicate the server is up and running.

**Comment by DrakeRichards** [comment]
Submitted PR https://github.com/vladmandic/automatic/pull/606 to re-add this feature.

**Comment by brknsoul** [comment]
Thanks everyone for this commit!

**Comment by w0lfram** [comment]
I am used to this feature in A1111, I would be very happy if you add it too.

**Reply by brknsoul** [reply]
Currently, the mp3 is pointed to `html\notification.mp3`. You can change this in Settings > Live Preview.

**Comment by vladmandic** [comment]
its been added and working quite some time ago.
